contracts/ETO/ETOTerms.sol

```diff
@@ -395,9 +406,12 @@ contract ETOTerms is
                 // discount is fixed so use base token price
                 if (discountedAmount > 0) {
                     // always round down when calculating tokens
-                    fixedSlotEquityTokenInt = discountedAmount / calculatePriceFraction(wlTicket.fullTokenPriceFrac);
-                    // todo: compute effective amount spent without the rounding
-                    // discountAmount = fixedSlotEquityTokenInt *  calculatePriceFraction(wlTicket.fullTokenPriceFrac);
+                    uint256 fixedSlotPrice = calculatePriceFraction(wlTicket.fullTokenPriceFrac);
+                    fixedSlotEquityTokenAmount = mul(discountedAmount, EQUITY_TOKEN_POWER) / fixedSlotPrice;
```

Comment:
why should this be higher than before?

Reply:
in other words we need to scale token amount to the TOKEN_POWER. and we multiply before we divide by price to minimize information lost by rounding down